src/components/collection-card/components/company-logo/index.tsx

```diff
@@ -3,22 +3,28 @@ import classNames from 'classnames'
 import { CompanyLogoOption, CompanyLogoProps } from './types'
 import s from './company-logo.module.css'
 
+const LOGO_TO_SOURCE_MAP: Record<CompanyLogoOption, string> = {
```

Comment:
I've moved `CompanyLogoOption` into `learn-client/types`. This felt more natural when constructing the `ProductPage` block types, and it felt intuitive to think of our component as filling out its logo options based on possibilities defined by `learn-client`, rather than the other way around.

On a related note, I've set up `LOGO_TO_SOURCE_MAP` for a similar reason - the intent is better type safety, as we'll now get a warning if `CompanyLogoOption` changes but we haven't account for it with an explicit image path.